templates/default/InstallChocolatey.ps1.erb

```diff
@@ -47,6 +47,16 @@ $tempDir = Join-Path $chocTempDir "chocInstall"
 if (![System.IO.Directory]::Exists($tempDir)) {[System.IO.Directory]::CreateDirectory($tempDir)}
 $file = Join-Path $tempDir "chocolatey.zip"
 
+try {
+  # This should work in .NET 4 where .NET 4.5 is installed as an inplace upgrade
+  # Enable TLS1.2 (3072)
+  [System.Net.ServicePointManager]::SecurityProtocol |= 3072
+  # Enable TLS1.1 (768)
+  [System.Net.ServicePointManager]::SecurityProtocol |= 768
+} catch {
+  Write-Warning "Unable to set PowerShell to use TLS 1.2 and TLS 1.1 due to old .NET Framework installed. Please upgrade to at least .NET Framework 4.5 and PowerShell v3 for this to work appropriately."
+}
```

Comment:
As an FYI, I would like the code to match as much as possible. https://github.com/chocolatey/chocolatey.org/blob/0994aa7d6f8942842917bc1bdf607d86cc504a28/chocolatey/Website/Install.ps1#L76-L88